### DISCUSSION TITLE
Get a better warning / more info when validating templates

### DISCUSSION THREAD

**Discussion by yzoug** [projectdiscovery/nuclei#6440] [General]
Hello, I run into this error message when trying to validate my templates:

https://github.com/projectdiscovery/nuclei/blob/9e9a4a82dadc76849758e1f7d89d55c8f86cf847/internal/runner/runner.go#L436

Nothing else. Is there a way to display more information (i.e. the template that fails validation, and at best why it fails)? I've also tried with verbose output:

```
$ nuclei -u https://www.TARGET -w workflow.yaml -duc -ev -v -validate

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.0.4

		projectdiscovery.io

[VER] Started metrics server at localhost:9092
[FTL] Could not validate templates: encountered errors while performing template validation
```

This should probably be updated to give a more meaningful error message.

Thanks!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
